File: wallendar-app/app/src/main/java/com/td/wallendar/addmembers/MembersViewHolder.java
package com.td.wallendar.addmembers;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.td.wallendar.R;

public class MembersViewHolder extends RecyclerView.ViewHolder {
    public MembersViewHolder(@NonNull View itemView) {
        super(itemView);
    }

    // TODO agregar para remover email y setear listener etc etc
    public void bind(final String memberEmail) {
        final TextView textView = itemView.findViewById(R.id.member_text_mail);

        textView.setText(memberEmail);
    }
}

File: wallendar-app/app/src/main/java/com/td/wallendar/addmembers/MembersViewHolder.kt
package com.td.wallendar.addmembers

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.td.wallendar.R

class MembersViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    // TODO agregar para remover email y setear listener etc etc
    fun bind(memberEmail: String) {
        val textView = itemView.findViewById<TextView?>(R.id.member_text_mail)
        textView.text = memberEmail
    }
}